src/wallet/asyncrpcoperation_common.cpp

```diff
@@ -202,11 +202,14 @@ void ThrowInputSelectionError(
             switch (err.side) {
                 case ActionSide::Input:
                     side = "inputs";
+                    break;
```

Comment:
I think this will conflict with #6653 which has merged.

Reply:
Ah, right – I thought it would unify, but the `;` removal from the other commit in this PR conflicts.